valhalla/thor/edgestatus.h

```diff
@@ -70,22 +72,26 @@ class EdgeStatus {
 
   /**
    * Set the status of a directed edge given its GraphId.
-   * @param  edgeid   GraphId of the directed edge to set.
-   * @param  set      Label set for this directed edge.
-   * @param  index    Index of the edge label.
-   * @param  tile     Graph tile of the directed edge.
+   * @param  edgeid      GraphId of the directed edge to set.
+   * @param  set         Label set for this directed edge.
+   * @param  index       Index of the edge label.
+   * @param  tile        Graph tile of the directed edge.
+   * @param  path_id     Identifies which path the edge status belongs to when tracking multiple paths
```

Comment:
basically all the functions here now have an optional path_id which gets bitwise or'd into the spare bits of the tile/level id so we can use the same edgestatus to track up to 127 paths